source/agora/flash/UpdateSigner.d

```diff
@@ -273,16 +297,23 @@ public class UpdateSigner
             break;
         }
 
+        writefln("%s: Settlement signature %s from %s received",
+            this.kp.V.prettify, seq_id, this.peer_pk.prettify);
+
         if (auto error = this.isInvalidSettleMultiSig(this.pending_settle,
             settle_res.value, priv_nonce, peer_nonce))
         {
             // todo: inform? ban?
-            writefln("%s: Error during validation: %s. For settle signature "
+            writefln("%s: Error durin g validation: %s. For settle signature "
```

Comment:
I think this is a typo.

Reply:
Yeah there's going to be lots of these here. But thanks.